portugues/colaborando.md

```diff
@@ -63,3 +63,5 @@ BR BR BR Hue hue hue!!!!
 [Corrigiria o bug fazendo o seguinte] (jc/jc.md)
 
 [Usaria: git clone, branch, add, commit e push] (https://www.google.com.br/)
+
+[Crio um novo branch e faço alterações no codigo do aplicativo, realizando commits,e uso o github para obter ajuda] (http://gitref.org/)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.